dev-cpp/yaml_cpp/yaml_cpp-0.6.2.recipe

```diff
@@ -30,6 +30,9 @@ PROVIDES_devel="
 REQUIRES_devel="
 	yaml_cpp$secondaryArchSuffix == $portVersion base
 	"
+CONFLICTS_devel="
+	yaml_cpp5${secondaryArchSuffix}_devel
```

Comment:
The conflicts in `yaml_cpp5-0.5.3.recipe` is enough and you can remove the one in this recipe.
```diff
-CONFLICTS_devel="
-	yaml_cpp5${secondaryArchSuffix}_devel
-	"
```
(I learned this with @korli.)